hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
+			sec, _ := strconv.ParseInt(time_pieces[0], 10, 64)
```

Comment:
I don't know enough about XMPP to dispute you, and I totally agree with error handling best practices (you're right), but I'm guessing that block never gets executed. It looks like XMPP always puts a time there.